agent/command/timeout.go

```diff
@@ -66,8 +66,13 @@ func (c *timeout) Execute(ctx context.Context, _ client.Communicator, logger cli
 		logger.Execution().Infof("Set idle timeout to %d seconds.", c.TimeoutSecs)
 	}
 	if c.ExecTimeoutSecs != 0 {
-		conf.SetExecTimeout(c.ExecTimeoutSecs)
-		logger.Execution().Infof("Set exec timeout to %d seconds.", c.ExecTimeoutSecs)
+		if conf.MaxExecTimeoutSecs > 0 && c.ExecTimeoutSecs > conf.MaxExecTimeoutSecs {
+			conf.SetExecTimeout(conf.MaxExecTimeoutSecs)
+			logger.Task().Errorf("Exec timeout %d seconds exceeds the limit of %d seconds. Set exec timeout to %d seconds.", c.ExecTimeoutSecs, conf.MaxExecTimeoutSecs, conf.MaxExecTimeoutSecs)
```

Comment:
nit: maybe `Warningf` instead since the timeout.update command continues on error. Using warning will also make the tet show up as yellow instead of red.